src/player.js

```diff
@@ -1132,6 +1199,7 @@ export default class Player extends FakeEventTarget {
     this._activeTextCues = [];
     this._tracks = [];
     this._firstPlay = true;
+    this._firstPlayInCurrentSession = false;
```

Comment:
@dan-ziv wouldn't this reset on change media?

Reply:
@OrenMe 
yes, to false - this is exactly what we want - the is not the first play in current session now